### PR TITLE
feat:Product list page & all filters added

### PR DESCRIPTION
**Pull request by rajkishancode**

### CODE REVIEW

File: screens/product/product.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  top: 8.8rem;
+  bottom: 0;
+  height: calc(100vh - 8.8rem);
+  overflow-y: auto;
+}
+
+.filter-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
```
Comment (aniketsingh13) [inline screens/product/product.css]: you can make  class for it. since, it repeat on your code

File: screens/product/product.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+.price-range {
+  display: flex;
+  justify-content: space-between;
```
Comment (aniketsingh13) [inline screens/product/product.css]: here also

File: screens/product/product.html [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+                            </span>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+
+
+
```
Comment (aniketsingh13) [inline screens/product/product.html]: remove unnecessary space in your code

File: screens/product/product.html [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+                <li class="my-2">
+                    <input class="mr-2" type="radio" name="price-sort" id="price-high-to-low" />
+                    <label for="price-high-to-low"> Price - high to low</label>
+                </li>
+            </ul>
+        </section>
+    </aside>
+
+
+
```
Comment (aniketsingh13) [inline screens/product/product.html]: here also

File: screens/product/product.html [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+                <div class="card-body p-4">
+                    <div class="card-title mb-2 text-center">
+                        Printed Hoodie
+                    </div>
+
+                    <div class="card-title mb-2 text-center">
+                        Rs. 2,500 <span class="ecom-original-price px-2"> Rs. 5,000 </span> <span class="ecom-discount">
+                            (50% OFF) </span>
+                    </div>
+
```
Comment (aniketsingh13) [inline screens/product/product.html:307]: here also

**Review by aniketsingh13 — APPROVED** [review]
add a screen shot and link so that it easy to review. remove the unnecessary space in your code . other things look good to me

File: screens/product/product.css (hunk 1 above)
Comment (rajkishancode) [reply screens/product/product.css]: okay i will refactor.

File: screens/product/product.css (hunk 2 above)
Comment (rajkishancode) [reply screens/product/product.css]: Done

**Review by aurobindoGupta — APPROVED** [review]

File: screens/product/product.css (hunk 1 above)
Comment (rajkishancode) [reply screens/product/product.css]: Done

**Pull request merged**